modules/ROOT/pages/index.adoc

```diff
@@ -55,7 +55,7 @@ OpenSSL is a required dependency for the driver.
 
 === Install OpenSSL
 
-. Download `Win64OpenSSL_Light-1_1_1v.msi` from the https://slproweb.com/products/Win32OpenSSL.html[Shining Light Productions site].
+. Download `Win64 OpenSSL v3.3.x Light` from the https://slproweb.com/products/Win32OpenSSL.html[Shining Light Productions site].
```

Comment:
It should work with v3.3.x but QE has signed off with v3.3.1. Should we say this version is for [couchbase-odbc-1.1-win64.msi](https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-1.1-win64.msi)?

Reply:
Hold on